ext/date/php_date.c

```diff
@@ -5255,34 +5253,6 @@ PHP_METHOD(DatePeriod, __wakeup)
 }
 /* }}} */
 
-/* {{{ date_period_read_property */
-static zval *date_period_read_property(zval *object, zval *member, int type, void **cache_slot, zval *rv)
```

Comment:
There are some forward declarations that should also be removed.

Reply:
Lines 663-664? I've removed those in 1c88abbc5ca4ee82298e560a34ad9b1e1c27fcae, thanks :+1: